models/bert_cve_classifier_torch.py

```diff
@@ -0,0 +1,84 @@
+import logging
+import math
+
+import daiquiri
+import numpy as np
+import pandas as pd
+import torch
+from torch import Tensor
+from torch.utils.data import TensorDataset, DataLoader
+from tqdm import tqdm
+from transformers import BertConfig, BertTokenizer
+from models.bert_model_weighted import BertForSequenceClassification
+from transformers.data.processors import (
+    glue_processors as processors,
+    glue_output_modes as output_modes,
+)
+
+from utils import model_constants as mc
+
+daiquiri.setup(level=logging.DEBUG)
```

Comment:
I did that intentionally, because in some places I needed debug and elsewhere it would be to much. If we read from config file then it would not be possible to do that. For example if I set the logging level to DEBUG in storage_utils the boto logs will fill up container.